.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   base:
     docker:
-      - image: cimg/base:2021.02
+      - image: cimg/base:stable-18.04
```

Comment:
The year/month is intentional as the most deterministic, isn't it possible to use the same type but newer month ?

Reply:
i.e `2021.11-18.04`

Reply:
Trouble with that is that it's intentionally broken every so often. Surely better to actually keep up with patches as our users do

Reply:
I don't mind to keep up with patches, but unintentional updates has been problem to only add additional hassle and always happen exactly when you don't have time for it :)

However, you have 2 approvals feel free to merge it as is :)